### PR TITLE
Upgrade role testing to Molecule 2

### PR DESCRIPTION
**Pull request by sbesson**
See https://github.com/ome/ansible-roles/issues/9

### CODE REVIEW

File: defaults/main.yml [hunk 1]
```diff
@@ -1,7 +1,8 @@
 ---
 # defaults file for postgresql-backup
 
-#postgresql_backup_dir:
-postgresql_backup_filename_format: "{{ ansible_hostname }}-%Y%m%d-%H%M%S.pgdump"
+# postgresql_backup_dir:
+postgresql_backup_filename_format: >-
+  "{{ ansible_hostname }}-%Y%m%d-%H%M%S.pgdump"
```
Comment (manics) [inline defaults/main.yml]: Need to check whether `"` inside a YAML multi-line string will end up as part of the output filename

Comment (sbesson) [reply defaults/main.yml]: I wondered about this as well but I expected https://github.com/ome/ansible-role-postgresql-backup/pull/1/files#diff-2bf9f9b5e1fff4f3458c5348395b319fL12 to fail if the quotes were preserved. I can try and remove them if this works and is the recommended style.

Comment (manics) [reply defaults/main.yml]: That tests a custom value `postgresql_backup_filename_format: "database-%Y%m%d.pgdump"`, not the default value

Comment (sbesson) [reply defaults/main.yml]: Fixed in 32c2d89. Also updated the molecule playbook to test a similar configuration

**Review by manics — APPROVED** [review]

**Pull request merged**